src/tep_sup.erl

```diff
@@ -24,14 +31,20 @@ start_link() ->
 %%                  shutdown => shutdown(), % optional
 %%                  type => worker(),       % optional
 %%                  modules => modules()}   % optional
-init([]) ->
+init([ConfigPath]) ->
     SupFlags = #{strategy => one_for_all, intensity => 0, period => 1},
+
+    Configuration = #{
+        id => configuration,
+        start => {configuration_server, start_link, [ConfigPath]}
+    },
+
     Storage = #{
         id => storage_sup,
         start => {storage_sup, start_link, []},
         shutdown => infinity,
         type => supervisor
     },
-    {ok, {SupFlags, [Storage]}}.
+    {ok, {SupFlags, [Configuration, Storage]}}.
```

Comment:
Can't find the tests that require this behaviour? Can we not reasonably test this?

Reply:
Since this would require changing the test every time we change something here, this is the same as the configuration file that you remarked above. Also, there is absolutely no logic in this. Whats your opinion on this?